src/pages/Hacker/Checkin.tsx

```diff
@@ -131,9 +63,9 @@ class CheckinContainer extends React.Component<{}, ICheckinState> {
    * Function which checks in a hacker as long as the provided data is of the correct format.
    * @param data Hacker ID, or the url to VIEW_HACKER_PAGE, such as 5c0dd463d95414ef5efd14cd, or https://app.mchacks.ca/application/view/5c0dd463d95414ef5efd14cd
    */
-  private async handleScan(data: string | null) {
-    if (data && !this.state.loading && this.state.lastScan !== data) {
-      this.setState({ lastScan: data });
+  const handleScan = async (data: string | null) => {
```

Comment:
why are we switching from `private`?

Reply:
Might be wrong, but don't think we can use `private` outside of a class?
Also since its inside of a function instead of a class now, it's inherently inaccessible by external code.